src/main/java/org/folio/rest/impl/PostGobiOrdersHelper.java

```diff
@@ -62,7 +62,8 @@
   public static final String CQL_CODE_STRING_FMT = "code==\"%s\"";
   public static final String TENANT_HEADER = "X-Okapi-Tenant";
   private static final String EXCEPTION_CALLING_ENDPOINT_MSG = "Exception calling {} {}";
-  private static final String DEFAULT_LOCATION_CODE = "*";
+  private static final String DEFAULT_LOOKUP_CODE = "*";
+  private static final String UNSPECIFIED_MATERIAL_ID = "unspecified";
```

Comment:
the name of this variable is misleading. It's not an ID...

Reply:
Renamed